cds-snc/cppd-medical-report-sails
# MAs can download supporting documents

# What this does

On the MA View of an application, adds a link to the Supporting Document(s).

This does not currently perform any validation or authorization. Simply enables downloading a document by id.

## To test
- If there are no already submitted applications, you may need to create one. 
- Make sure to upload some supporting documents and link them to at least one Condition.
- Navigate to the MA interface at `/en/sessions`
- Open an application
- If there are no Supporting Documents, try another application
- If there are Supporting Documents, try clicking on one!

Note that since the filesystem of our staging environments are ephemeral, the document *may* no longer exist, and in that case you should see a 400 error, and that's expected. If you get a 500 error, then something else is wrong.
